core/src/main/resources/db/migration/V21__add_campaigns.sql

```diff
@@ -0,0 +1,17 @@
+create table `Campaign` (
+`uuid` char(36) NOT NULL,
+`namespace` char(200) NOT NULL,
```

Comment:
you should use package_uuids for instead of `(namespace, version, name)` (rebase master). And have a foreign key to Packages using that field.